src/playbook.yaml

```diff
@@ -2,6 +2,33 @@
   become: yes
 
   tasks:
-  - name: Print debug message
-    debug:
-      msg: Hello, world!
+
+  - name: Add Docker GPG key
+    apt_key: url=https://download.docker.com/linux/ubuntu/gpg
+
+  - name: Add Docker APT repository
+    apt_repository:
+      repo: deb [arch=amd64] https://download.docker.com/linux/{{ ansible_distribution|lower }} {{ansible_distribution_release}} stable
+      update_cache: yes
+
+  - name: install packages
+    package:
+      name:
+        - python3-pip
+        - docker-ce
+        - docker-ce-cli
+        - containerd.io
+      state: present
+
+  - name: Docker post-install
+    shell: |
+      if [ -z $(getent group docker) ]; then
+        groupadd docker
+      fi
+      gpasswd -a {{ lookup('env','SSH_USERNAME') }} docker
```

Comment:
Will this dynamically look up the user upon creation of the new instance.   Or will the user have to add themselves since the only user are adding is the SSH_USERNAME you are passing in.

Reply:
No, planning to do that in Cloudformation template where the JC association is done. I'll have the JC ID there. I thought about moving this logic there as well, but sort of prefer to put it here so that the AMI is ready to go outside of the whole CFN and SC setup.

Reply:
A service catalog user create a workflows resource, sshes into the resource, switches user to ubuntu, and runs the workflows as the ubuntu user? That works ok, but is really for single user systems. It would be better to stick with jumpcloud users and and the local user installed by Jumpcloud to the right unix groups.